src/communication/xcConnection.ts

```diff
@@ -34,7 +29,9 @@ export class DefaultConnection implements Connection {
             log.debug("getModel request failed");
             log.debug(err);
         };
-        session.init(openListener, errorListener);
+        let closeListener = (_: CloseEvent) => {
```

Comment:
For the moment the errors are just in the logs. But we could add an error parameter in the Listener to make sure that there wasn't errors.